### ISSUE TITLE
Make logos in header and footer not background images

### ISSUE THREAD

**Issue by eelzi-ONRR** [DOI-ONRR/onrr.gov-site#3245]
Found via Andi - the ONRR logos in the header and footer are background images. Only decorative images can be background images and logos are not considered decorative. Violates WCAG 1.1.1

<img width="1920" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeHd0Qmc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--a1b72811b885117d2b29c3deb082d66d30c6eed8/image.png" alt="image.png" />
